tasks/main.yml

```diff
@@ -38,6 +38,7 @@
     - "{{ prometheus_rules_dir }}"
     - "{{ prometheus_snmp_exporter_config_dir }}"
     - "{{ prometheus_blackbox_exporter_config_dir }}"
+  when: prometheus_install
```

Comment:
Let's say only `alert manager` or `snmp exporter` or `blackbox exporter` is installed this will fail. Ideally, each install-*.yml file should have this block.